config/config/common-config.yaml

```diff
@@ -0,0 +1,370 @@
+## Common Kubernetes configuration options.
+## This configuration applies to both service and workload clusters.
+
+## Define resources requests and limits for single Pods.
+## ref: https://kubernetes.io/docs/user-guide/compute-resources/
+## resources: {}
+
+## Node labels for Pod assignment
+## ref: https://kubernetes.io/docs/user-guide/node-selection/
+## nodeSelector: {}
+
+## Tolerations for Pod assignment
+## ref: https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/
+## tolerations: []
+
+## Affinity for Pod assignment
+## ref: https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#affinity-and-anti-affinity
+## affinity: {}
+
+## Some common options used in various helm charts.
+##
+global:
+  ## Compliantkubernetes-apps version.
+  ## Use version number if you are exactly at a release tag.
+  ## Otherwise use full commit hash of current commit.
+  ## 'any', can be used to disable this validation.
+  ck8sVersion: ${CK8S_VERSION}
+  ck8sCloudProvider: ${CK8S_CLOUD_PROVIDER}
+  ck8sEnvironmentName: ${CK8S_ENVIRONMENT_NAME}
+  ck8sFlavor: ${CK8S_FLAVOR}
```

Comment:
Do we need this variable? Isn't this only used for the first init to decide on default values?

Reply:
As `init` now also can be used to update the default configs by pulling values from the template configs from the repo, this is needed to ensure that these values are not inadvertently changed.
So we basically can't change the config from a Safespring production cluster to an Exoscale dev without warnings if an env is set wrong or missed.

Reply:
Valid argument. Then I'm fine with this :+1: